scripts/git.mjs

```diff
@@ -0,0 +1,65 @@
+import {gitTag} from '@coveo/semantic-monorepo-tools';
+import {readFileSync} from 'fs';
+import {resolve} from 'path';
+import {execute} from './exec.mjs';
+
+/**
+ * @typedef {import('./packages.mjs').PackageDefinition} PackageDefinition
+ */
+
+const releaseBranch = 'master';
+
+async function getBranchName() {
+  return await execute('git', ['branch', '--show-current']);
+}
+
+export async function isOnReleaseBranch() {
+  return (await getBranchName()) === releaseBranch;
+}
+
+export async function getHowManyCommitsBehindUpstream() {
+  return parseInt(await execute('git', ['rev-list', '--count', 'HEAD..@{u}']));
+}
+
+export async function getHeadCommitTag() {
+  return await execute('git', ['tag', '--points-at', 'HEAD']);
+}
+
+/**
+ * @param {string} tag
+ */
+export async function tagExists(tag) {
+  return !!(await execute('git', ['tag', '-l', tag]));
+}
+
+export async function stageAll() {
+  await execute('git', ['add', '.']);
+}
```

Comment:
I reckon there's some stuff that may be available or could be available in semantic-monorepo-tools

Reply:
replacing `stageAll` with `gitAdd`